wagtail/admin/templates/wagtailadmin/home/pages_for_moderation.html

```diff
@@ -1,75 +1,75 @@
 {% load i18n wagtailadmin_tags %}
 {% if page_revisions_for_moderation %}
-    <div class="panel nice-padding">{# TODO try moving these classes onto the section tag #}
-        <section>
-            <h2>{% trans 'Pages awaiting moderation' %}</h2>
-            <table class="listing">
-                <col />
-                <col width="30%"/>
-                <col width="15%"/>
-                <col width="15%"/>
-                <thead>
+<section class="object collapsible">
+    <h2 class="title-wrapper">{% trans 'Pages awaiting moderation' %}</h2>
+    <div class="object-layout">
+        <table class="listing">
+            <col />
+            <col width="30%"/>
+            <col width="15%"/>
+            <col width="15%"/>
+            <thead>{# Note: the header is visually hidden behind .title-wrapper #}
```

Comment:
I'm not sure about this. For some - edited pages - it seems fine, but for the workflow icons, the circle floating without contextual information and the "x minutes ago" referring to different things (edits, tasks started, locked) in different tables seems unintuitive to me. Happy to be overruled on this since it's design/UX, but wonder if it's worth checking this over with them?

Reply:
This was specifically requested as part of that process. Let's take it offline with Dex/Ben E

Reply:
Sounds good. As I say, happy to be overruled here, I'm not a designer!